packages/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.5.20+1
```

Comment:
Since we are adding a new feature to the plugin, we would need to update the minor version: 0.5.21